math/ncr_modulo_p.cpp

```diff
@@ -25,71 +25,85 @@ namespace math {
  * implementation.
  */
 namespace ncr_modulo_p {
+
+namespace utils {
+/** Finds the value of x, y such that a*x + b*y = gcd(a,b)
+ *
+ * @params[in] the numbers 'a', 'b' and address of 'x' and 'y' from above
+ * equation
+ * @returns the gcd of a and b
+ */
```

Comment:
please follow documentation standards when documenting functions; i.e add @brief tag and split the parameters into their own tags i.e @param a <description> @param b <description>